### DISCUSSION TITLE
Setting Hotkeys Blank

### DISCUSSION THREAD

**Discussion by craigccfl** [material-shell/material-shell#840] [Q&A]
Coming from 2-years Material-Awesome, loving Material-Shell on Fedora 36. Wondering if anyone knew how to get the hotkeys section to list. Mine is blank. I know it is something I did because it was working till I started playing with things. BTW, the hotkeys are working, just have to go to the website for training. 

![image](https://user-images.githubusercontent.com/38798834/175790941-c8bc1782-b477-4ee9-bada-b9b4554d6f21.png)

**Comment by craigccfl** [comment]
Resolved by uninstalling material-shell and reinstalling. 
![image](https://user-images.githubusercontent.com/38798834/179919198-f791795b-1a88-4e0b-8b15-fdbc071bebd7.png)

**Reply by PapyElGringo** [reply]
It's because we have landed a new release I believe 😁
